Uniswap/interface
# feat: update cancelled local tx's

## Description


Updates activity feed to combine local and remote data to display more information about cancelled transactions, rather than just showing an empty cancel tx


_Linear ticket:_ https://linear.app/uniswap/issue/WEB-2064/[mp]-display-activity-with-cancelled-status


## Screen capture

https://github.com/Uniswap/interface/assets/39385577/50aa58aa-3263-4f22-8574-39fbaffb5807

|    Before    |   After    |
| ------------ | ------------ |
| <img width="406" alt="image" src="https://github.com/Uniswap/interface/assets/39385577/53f2cc92-2db4-4ae5-8f17-95090a9254ba"> | <img width="411" alt="image" src="https://github.com/Uniswap/interface/assets/39385577/4b825b0e-0647-4c31-b287-c4f31cdceaba"> |


## Test plan

### QA (ie manual testing)


- Submit a transaction
- Cancel it in your wallet
- Open activity history (or refresh if you already have it opened) and verify that "cancel" is displayed, instead of a stuck pending tx
